packages/vscode-extension/src/project/metro.ts

```diff
@@ -515,19 +526,25 @@ export class MetroLauncher extends Metro implements Disposable {
               this.bundleProgressEventEmitter.fire({ bundleProgress });
             }
           } else if (event.type === "client_log" && event.level === "error") {
-            Logger.error(stripAnsi(event.data[0]));
+            const err = stripAnsi(event.data[0]);
+            Logger.error(err);
+            metroOutputChannel?.appendLine(err);
           } else {
             Logger.debug("Metro", line);
           }
 
+          let log;
+
           switch (event.type) {
             case "RNIDE_expo_env_prelude_lines":
               this._expoPreludeLineCount = event.lineCount;
               Logger.debug("Expo prelude line offset was set to: ", this._expoPreludeLineCount);
               break;
             case "initialize_done":
               this._port = event.port;
-              Logger.info(`Metro started on port ${this._port}`);
+              log = `Metro started on port ${this._port}`;
+              metroOutputChannel?.appendLine(log);
```

Comment:
wouldn't it be better if we also used log levels here instead of `appendLine` ? Seems like that would be more consistent with other output channel that we use:
```
metroOutputChannel.info(log);
```

Reply:
All the other places where we append to channel are where we directly want to pipe output of another process. Here, we create log entries on our own, so I wouldn't compare those two cases. This one is more similar to the Logger case, we just need the log to appear in another channel

Either way, I'm ok keeping this as is. Doesn't seem too imporant for the whole PR to be blocked on the log formatting output.